trajopt_optimizers/trajopt_optimizers_test_suite/include/basic_optimization_unit.hpp

```diff
@@ -0,0 +1,187 @@
+#include <trajopt_utils/macros.h>
```

Comment:
Add a license notice. I'm not sure how heavily this is based on the IFOPT examples, but we should probably credit the author here and make this test BSD3 licensed.